src/inc/msquic.hpp

```diff
@@ -802,6 +814,11 @@ struct MsQuicConnection {
                 &Value);
     }
 
+    QUIC_STATUS
+    SetKeepAlive(_In_ uint32_t Value) noexcept {
+        return SetSettings(MsQuicSettings{}.SetKeepAlive(Value));
+    }
+
```

Comment:
I'd rather not add this helper to msquic.hpp. If we follow this pattern to the logical conclusion, now you have a ton of duplication everywhere (i.e. add all settings here; then add it to the listeners, etc.). This is simple enough to put at the caller instead.